adagrams/game.py

```diff
@@ -1,13 +1,79 @@
 from random import randint
 
+LETTERS_DICT ={
```

Comment:
Operators like `=` should be surrounded with white spaces. Take a look at the [PEP8 style guide on white spaces](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements).  In industry, code that doesn't follow an agreed upon code styles will need to be revised to ensure the codebase is neat and consistent.
```suggestion
LETTERS_DICT = {
```